ui/job-view/headerbars/SecondaryNavBar.jsx

```diff
@@ -283,6 +282,7 @@ class SecondaryNavBar extends React.PureComponent {
               } ${!duplicateJobsVisible ? 'strikethrough' : ''}`}
               tabIndex="0"
               role="button"
+              aria-label="Show/Hide duplicate jobs"
```

Comment:
I think it'd be clearer for the user of the screen reader to have this text use the same logic as the title.

Reply:
Okay, I will change that.

Reply:
Can I do this too?
```jsx
              aria-label={
                duplicateJobsVisible
                  ? 'Hide duplicate jobs'
                  : 'Show duplicate jobs'
              }
```
I was looking for a way both the title and aria-label attribute could share the same value, but I have no idea.

Reply:
Yes, that would work. But let's not duplicate the same code twice (keep it DRY).

Reply:
I need an idea on how to do this without writing it twice. Should I create a helper function?

Reply:
That is one approach but you don't need a function for a ternary operator - you can store it in a variable.

Reply:
@bolah2009 : A helper function would work.  But I would go with setting the values in a couple ``let``s at the top end of the ``render`` function.  Then if ``duplicateJobsVisible`` is true, reset the values.  Though @sarah-clements may have a different approach in mind.  There are several ways to do it, tbh.  :)

Reply:
Okay, thanks @camd :relaxed: